tests/app/submitter/test_converter.py

```diff
@@ -746,6 +746,904 @@ def test_converter_q_codes_for_empty_strings(self):
             self.assertEqual(len(answer_object['data']), 1)
             self.assertEqual(answer_object['data']['2'], 'Ready salted')
 
+    def test_radio_answer(self):
+        with self._app.test_request_context():
+            routing_path = [Location(group_id='radio-group', group_instance=0, block_id='radio-block')]
+            user_answer = [create_answer('radio-answer', 'Coffee', group_id='radio-group',
+                                         block_id='radio-block')]
+
+            questionnaire = {
+                "data_version": "0.0.1",
+                "survey_id": "0",
+                "groups": [
+                    {
+                        "id": "radio-group",
+                        "blocks": [
+                            {
+                                "id": "radio-block",
+                                "questions": [
+                                    {
+                                        "id": "radio-question",
+                                        "answers": [
+                                            {
+                                                "type": "Radio",
+                                                "id": "radio-answer",
+                                                "q_code": "1",
+                                                "options": [
+                                                    {
+                                                        "label": "Coffee",
+                                                        "value": "Coffee"
+                                                    },
+                                                    {
+                                                        "label": "Tea",
+                                                        "value": "Tea"
+                                                    }
+                                                ]
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            }
+
+            # When
+            answer_object = convert_answers(self.metadata, QuestionnaireSchema(questionnaire), AnswerStore(user_answer),
+                                            routing_path)
+
+            # Then
+            self.assertEqual(len(answer_object['data']), 1)
+            self.assertEqual(answer_object['data']['1'], 'Coffee')
+
+    def test_radio_answer_ver_2(self):
+        with self._app.test_request_context():
+            routing_path = [Location(group_id='radio-group', group_instance=0, block_id='radio-block')]
+            user_answer = [create_answer('radio-answer', 'Coffee', group_id='radio-group',
+                                         block_id='radio-block')]
```

Comment:
Minor nitpick why the new line? You can keep them on the same line in the same style as the rest of the file.